src/courses/profile-dev-test/11.md

```diff
@@ -1,20 +1,72 @@
 ---
 order: 11
 next: 12.md
-title: Rules of the Road
+title: 11. Understanding Profile Updates
 author: Aaron Lippold
 ---
 
-## Best Practices for Profile Managment
+## Learning Objectives
 
-When updating Benchmark Profiles, adhere to these key principles to maintain alignment with the original Guidance Documents:
+By the end of this section, you will be able to:
 
-1. **Maintain Version Integrity:** **Never Merge** new requirements into older benchmark branches. This will create a 'mixed baseline' that doesn't align with any specific guidance document. Benchmarks, STIGs, and Guidance Documents form a 'proper subset' - they should be treated as 'all or nothing'. Mixing requirements from different versions can invalidate the concept of 'testing to a known benchmark'.
+- Identify the three types of profile updates
+- Understand the scope of STIG and CIS Benchmark updates
+- Recognize the forward-only nature of security benchmark updates
 
-2. **Benchmarks are a Complete Set of Requirements:** A Security Benchmark is 'complete and valid' only when all requirements for a specific Release or Major Version are met. Unlike traditional software projects, features and capabilities cannot be incrementally added. A Security Benchmark and its corresponding InSpec Profile are valid only within the scope of a specific 'Release' of that Benchmark.
+## Types of Profile Updates
 
-3. **Release Readiness Is Predefined:** A Benchmark is considered 'ready for release' when it meets the expected thresholds, hardening, and validation results. Don't be overwhelmed by the multitude of changes across the files. Instead, focus on the specific requirement you are working on. Understand its expected failure and success states on each of the target testing platforms. This approach prevents you from being overwhelmed and provides solid pivot points as you work through the implementation of the automated tests for each requirement and its 'contexts'.
+Security benchmark profiles require regular updates to maintain their effectiveness. Let's explore the three main types of updates:
 
-4. **Use Vendor-Managed Standard Releases:** When setting up a test suite, prioritize using vendor-managed standard releases for software installations and baseline configurations. This should be the starting point for both 'vanilla' and 'hardening' workflows. This approach ensures that your initial and ongoing testing, hardening, and validation closely mirror the real-world usage scenarios of your end-users.
+<table>
+  <thead>
```

Comment:
Why isn't this a markdown table?

Reply:
cause markdown tables can't do headers on the left hand side properly